worker/instancepoller/machine_test.go

```diff
@@ -46,10 +55,10 @@ func (s *machineSuite) TestSetsInstanceInfoInitially(c *gc.C) {
 	}
 	died := make(chan machine)
 
-	clock := newTestClock()
-	go runMachine(context, m, nil, died, clock)
-	c.Assert(clock.WaitAdvance(LongPoll, coretesting.ShortWait, 1), jc.ErrorIsNil)
-	c.Assert(clock.WaitAdvance(LongPoll, coretesting.ShortWait, 1), jc.ErrorIsNil)
+	clk := newTestClock()
```

Comment:
This drive-by is in its own commit. View the fourth commit to see the functional change.